probe-rs-tools/src/bin/probe-rs/cmd/info.rs

```diff
@@ -440,21 +450,64 @@ fn coresight_component_tree(
     Ok(tree)
 }
 
+fn process_known_rom_tables(
```

Comment:
Can we add descriptions to those functions? Or maybe change this name? From the name it's not clear to me what it does. And when i read the code in serial, I would like to be able to just read or at least hover the function and get an explanation in editor without going to read the full function :)

Reply:
Lemme know how the last commit works for you.

Reply:
Looking great :)